### DISCUSSION TITLE
More compact boolean display using field name instead of "enabled"?

### DISCUSSION THREAD

**Discussion by markcharlesobrien** [directus/directus#14028] [Archive]
Hello, I am a new Directus user. I have a database with a lot of booleans. Even with the fields set to "half width" it seems to take up a lot of screen space to have the field label, then the frame around the check box, then the word "Enabled" and a check box. I was wondering if there was a way to make it more compact. For example have just the field name and a check box next to it.

**Comment by benhaynes** [comment]
Absolutely! This is one of the main edits I added in: https://github.com/directus/directus/discussions/9161

Of course, that mockup is just one way to enable a better toggle... but that thread is meant to be a starting point for broader ideas.

**Reply by markcharlesobrien** [reply]
Hi Ben, thank you for your reply. Maybe I'm not understanding - a mockup is like a Photoshop of what you'd like to see an interface do in the future, right? I was asking if there was a way to do this in Directus currently. As I am a new Directus user, I am perhaps just unaware of where certain options are.

**Reply by benhaynes** [reply]
The mockup is an improvement planned for the future. Currently, the toggle shows the field name and you can customize the label in settings. There is a way to "Hide the field name" but that is only in code. Of course, you can always create a custom interface to do what you're asking.

**Reply by markcharlesobrien** [reply]
From what research I've been able to do, if I had any idea how to write code I would have never run across Directus, so unless I'm misunderstanding, creating a custom interface is probably out of my ability. Coding custom database interfaces seems to be the norm, which is why I am enjoying Directus so much. I will keep an eye on your updates and I look forward to seeing any new display options you introduce in upcoming versions.
